core/lib/dal/migrations/20231213192041_applied-snapshot-status.up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE applied_snapshot_status
+(
+    l1_batch_number         BIGINT NOT NULL PRIMARY KEY,
+    is_finished             BOOL NOT NULL,
+    last_finished_chunk_id   INT
```

Comment:
We have discussed that we'd want to store snapshot metadata in this table, right? For the purposes of metadata calculator recovery, we'd need:

- Snapshot miniblock number
- Hash of the snapshot L1 batch

For the state keeper recovery, we'd most probably also need:

- Hash of the snapshot miniblock

Also, AFAIU, we have `created_at` / `updated_at` timestamps for all tables for audit purposes.

Reply:
I added all those columns